src/System.Collections/src/System/Collections/Generic/SortedSet.cs

```diff
@@ -1826,7 +1829,14 @@ public Node DeepClone(int count)
                 return newRoot;
             }
 
-            public int GetCount() => 1 + (Left?.GetCount() ?? 0) + (Right?.GetCount() ?? 0);
+            public int GetCount()
+            {
+#if !DEBUG
```

Comment:
We shouldn't even build it into release builds.  We should just wrap the whole function and any usage in #if DEBUG.